client/app.ts

```diff
@@ -0,0 +1,45 @@
+import 'reflect-metadata';
+import 'zone.js/dist/zone';
+import { Component, provide } from '@angular/core';
+import { bootstrap } from 'angular2-meteor-auto-bootstrap';
+import { ROUTER_PROVIDERS, ROUTER_DIRECTIVES, RouteConfig, RouterLink } from '@angular/router-deprecated'; 
+import { APP_BASE_HREF } from '@angular/common';
+import { Account } from './imports/account/account.ts';
+import { Assignment } from './imports/assignment/assignment.ts';
+import { Grade } from './imports/grade/grade.ts';
+import { Login } from './imports/login/login.ts';
+import { Home } from './imports/home/home.ts';
+import { CoursePage } from './imports/coursepage/coursepage.ts';
```

Comment:
You do not need to import every single module in the app.ts.  Angular is designed to accept a tree of includes, and so you would include the things specific to the CoursePage only within the CoursePage component.